src/core/core.scale.js

```diff
@@ -789,43 +791,53 @@ module.exports = Element.extend({
 
 				tx1 = xTickStart;
 				tx2 = xTickEnd;
-				x1 = chartArea.left;
-				x2 = chartArea.right + axisWidth;
-				ty1 = ty2 = y1 = y2 = yLineValue;
+				ty1 = ty2 = yLineValue;
 			}
 
 			itemsToDraw.push({
 				tx1: tx1,
 				ty1: ty1,
 				tx2: tx2,
 				ty2: ty2,
-				x1: x1,
-				y1: y1,
-				x2: x2,
-				y2: y2,
 				labelX: labelX,
 				labelY: labelY,
-				glWidth: lineWidth,
-				glColor: lineColor,
-				glBorderDash: borderDash,
-				glBorderDashOffset: borderDashOffset,
+				tmWidth: lineWidth,
```

Comment:
It took me awhile to figure out, but I'm guessing `tm` stands for tick mark. I would just use `tick` instead since it's not much longer and is much clearer